stdlib/database/h2/src/test/resources/ballerina.conf

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+connection.time.out=1000
```

Comment:
What is the purpose of this file? Are we using them for a test?

Reply:
Yes, this is used in SQLConnectionPoolTest class.

Reply:
IMO we need to think about a way to redesign the test to validate whether the parameters have taken from the file or not

Reply:
Actually, this is used not just to test whether the value is taken from the conf file but also to set the connection timeout to a low value to avoid the tests taking a lot of time. 

This value gets automatically tested in many tests when asserting the timeout log.